### ISSUE TITLE
RC2 Blazor Wasm with DotNetCore server using Windows Authentication - can't debug

### ISSUE THREAD

**Issue by elylv** [dotnet/aspnetcore#27437]
With an .NET5 RC2 Blazor WebAssembly project, with Windows Authentication (rather than Anonymous Authentication) enabled on the server.

When you attempt to debug the project, the browser will open, but the page doesn't load and eventually VS2019 reports:
_"One or more errors occurred.

Failed to launch debug adapter.  Additional information may be available in the output window.

Exception of type
'Microsoft.VisualStudio.Shared.VSCodeDebugProtocol.ProtocolException' was thrown."_

This means I cannot debug the application as it will be running in production (so can't access HttpContext.User windows authentication info).

I believe this was supposed to be fixed in RC1 https://github.com/dotnet/aspnetcore/issues/21466 but it still persists for me.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us. This is a known issue for 5.0. We've documented how to work around it [here](https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.1&tabs=visual-studio#troubleshoot) (see the last bullet point).

We will try to address the underlying issue in 6.0 and is being tracked as part of https://github.com/dotnet/aspnetcore/issues/27301

**Comment by elylv** [comment]
> Thanks for contacting us. This is a known issue for 5.0. We've documented how to work around it [here](https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.1&tabs=visual-studio#troubleshoot) (see the last bullet point).

I've added `"NO_PROXY": "localhost"` in the Environmental Variables section of launchSettings for IIS Express in both my Server and Client projects, but it doesn't seem to have made any difference.  Is this the correct thing I should be trying?  The bullet point you mentioned is a little vague.
I also tried adding a system environmental variable for NO_PROXY with its value as localhost, but still no luck.
